src/editor/editor.directive.ts

```diff
@@ -226,20 +224,24 @@ export class FroalaEditorDirective implements ControlValueAccessor {
 
     // init editor
     this.zone.runOutsideAngular(() => {
-      this._$element.on('froalaEditor.initialized', () => {
+      this.registerEvent(this._$element, "initialized", () => {
         this._editorInitialized = true;
-      })
+      });
 
-      this._editor = this._$element.froalaEditor(this._opts).data('froala.editor').$el;
```

Comment:
The `froalaEditor` returns the instance of the editor. Thus, it would be good to replace `this._editor` with the editor instance and where required,  use `this._editor.$el` instead of `this._editor`.